### PR TITLE
BLOCKCHAIN-200 - FE - Form for creating motions for `democracy.external_propose_majority` referendums for legislation changes.

### PR DESCRIPTION
**Pull request by MichalLustykNeti**

### CODE REVIEW

File: src/redux/sagas/congress.js [hunk 1]
```diff
@@ -289,12 +339,22 @@ export function* congressProposeLegislationWatcher() {
...
-  } catch (e) {
-    yield put(congressActions.congressRepealLegislation.failure(e));
-  }
+  yield* blockchainWatcher(
+    congressActions.congressRepealLegislation,
+    congressRepealLegislationWorker,
+  );
+}
+
+export function* congressProposeRepealLegislationWatcher() {
```
Comment (kacperzuk-neti) [inline src/redux/sagas/congress.js:348]: Seems to be unused.

File: src/components/Legislation/LegislationView/index.js [hunk 2]
```diff
@@ -69,6 +70,7 @@ const LegislationView = () => {
                 )
             }
             { tier === '1' && <RepealLegislationButton tier={tier} index={l.index} /> }
+            { Number(tier) >= '2' && <ProposeRepealLegislationButton tier={tier} index={l.index} /> }
```
[the remark this answers is not in the document]
Comment (MichalLustykNeti) [reply src/components/Legislation/LegislationView/index.js:73]: So should i change this statement to:

`umber(tier) >= 1`

[the remark this answers is not in the document]
Comment (kacperzuk-neti) [reply src/components/Legislation/LegislationView/index.js:73]: Yes

**Review by kacperzuk-neti — APPROVED** [review]

**Review by DorianSternVukotic — APPROVED** [review]

File: src/components/Modals/CongressRepealLegislationFastTrackModal.js [hunk 3]
```diff
@@ -0,0 +1,96 @@
...
+    >
+      <div className={styles.h3}>Propose a Congress Motion - propose referendum for legislation repeal</div>
+
+      <div className={styles.title}>Legislation Tier</div>
+      <SelectInput
+        register={register}
+        name="tier"
+        disabled
+        options={[
+          { value: '1', display: 'International Treaties' },
```
[the remark this answers is not in the document]
Comment (kacperzuk-neti) [reply src/components/Modals/CongressRepealLegislationFastTrackModal.js:42]: https://github.com/liberland/liberland_substrate/blob/main/frame/liberland-legislation/src/lib.rs#L111-L117
Yup, they match the blockchain

**Pull request merged**